### DISCUSSION TITLE
Drag and drop to the app window is disabled for some reason

### DISCUSSION THREAD

**Discussion by Dachiko** [receyuki/stable-diffusion-prompt-reader#49] [Q&A]
When I'm dragging the image (jpg) to the app, windows 11 doesn't allow me to drop it. What could be the reason?

**Comment by receyuki** [comment]
Sorry, I don’t often check Discussions. Does this issue still occur on v1.3.4?

**Reply by Dachiko** [reply]
At some point it resolved itself, don't remember if it was after update or before. I think we can safely close the issue, it doesn't look like a systematic problem. Thanks for the app btw!

**Comment by Dachiko** [comment]
Probably my local problem, got resolved by itself
